portal/src/main/webapp/WEB-INF/jsp/global/js_include.jsp

```diff
@@ -109,6 +109,9 @@
 <script type="text/javascript" src="js/src/plots-tab/util/stylesheet.js"></script>
 <script type="text/javascript" src="js/src/plots-tab/util/plotsUtil.js"></script>
 
+<%-- TODO: workaround: cytoscape.js needs to be included before virtual cohort session libraries --%>
+<script type="text/javascript" src="js/lib/cytoscape_js/cytoscape.js?<%=GlobalProperties.getAppVersion()%>"></script>
```

Comment:
then do you need to remove a script tag somewhere else?

Reply:
Ideally yes, but there is also the `js_include_standard.jsp` where we don't include all the libraries, so we may still need the other script tag in its original place (`networks.jsp`).

I also noticed that virtual cohort session libraries in `js_include_standard.jsp` are included without the `$.noConflict()` call. So, actually, it is pretty confusing...